content/post/value-reordering-lists.md

```diff
@@ -0,0 +1,202 @@
+
+![Impact of FDNY's RBIS predictive inspection program, via Drew Conway](FDNY-RBIS-DC.png)
+
+The AUC metric associated with ROC curves like the above can be 
+interpreted as a rank-based metric: "the probability that a classifier will rank a randomly chosen positive instance higher than a randomly chosen negative one." 
+([via Wikipedia](https://en.wikipedia.org/wiki/Receiver_operating_characteristic))
+A similar, and even simpler-to-interpret visualization, the decile-based 
+lift curve, has been used for 
+decades by predictive analytics professionals. Typically, after ordering the test
+set by likelihood, you divide the list into tenths. For each tenth, you calculate the ratio of
+the number of 
+positive examples compared to the number you'd get if you
+sorted randomly instead. The ugly lift chart below would tell you that you'll be three times
+more effective if you use a predictive model versus a naive approach. That's impact!
+
+![Lift chart example](ugly-lift-chart.png)
+
+# Reordering and Optimization
+
+The FDNY building-inspection problem can also illustrate the next level of
+complexity. It's great that we have a ranked list of
+locations to inspect, in decreasing risk of fire. But what if that list happened
+to be ordered such that locations 1, 3, and 5 were on the far West side of town, and 
+locations
+2, 4, and 6 were on the far East side? We'd be wasting a great deal of time criss-crossing,
+which could put buildings at risk longer than they'd need to be, defeating the 
+purpose of our powerful model.
+Perhaps it would be better to take travel-time into consideration?
+
+This sort of optimization problem has been tackled for many decades by
+the field of operations research (OR), theoretical computer science, and applied math. 
+A classic example is the 
+[traveling salesperson problem](https://en.wikipedia.org/wiki/Travelling_salesman_problem),
+where the goal is to
+order a list of cities to minimize total travel distance. In the FDNY example, we have
+a 
+[multi-objective optimization](https://en.wikipedia.org/wiki/Multi-objective_optimization) 
+problem, where geographic optimization is combined with a ranking problem. 
+If we put a _cost_ on inspecting locations in the wrong order, and a _cost_ on
+driving, we can numerically optimize the order such that the overall cost is minimized. 
+By using techniques such as 
+[integer programming](https://en.wikipedia.org/wiki/Integer_programming) or
+[genetic algorithms](https://en.wikipedia.org/wiki/Genetic_algorithm), we might find
+orderings of the property list that solve the problem more efficiently
+
+# Presenting Ordered Lists
+
+In the discussions above, I've implied that the user is seeing a list -- of 
+ranked sales leads, or fire-prone properties, or whatever. But what choices can be made in
+the visual design of that list, so that users  
+interpret the ranking appropriately, and trust that the ranking is relevant?
```

Comment:
Did you mean to put this line break after `so that users`?

Reply:
it's Markdown -- don't think it matters...